src/main/java/frc/robot/subsystems/intake/IntakeMoveToAngle.java

```diff
@@ -0,0 +1,54 @@
+
+package frc.robot.subsystems.intake;
+
+import edu.wpi.first.math.trajectory.TrapezoidProfile;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.Command;
+
+public class IntakeMoveToAngle extends Command {
+  private final Intake intake;
+ private final TrapezoidProfile trapezoidProfile = new TrapezoidProfile(new TrapezoidProfile.Constraints(
+  IntakeConstants.MAX_VELOCITY,
+  IntakeConstants.MAX_ACCELERATION));
+  
+private final Timer timer = new Timer();
+
+  public IntakeMoveToAngle(Intake intake, double angle){
+    this.intake = intake;
+
+    // Use addRequirements() here to declare subsystem dependencies.
+    addRequirements(intake);
+    
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    timer.restart();
+    timer.start();
```

Comment:
`restart()` is already starting the timer, you don't need to call `start()` again